testing/sdk_networks.py

```diff
@@ -19,3 +27,23 @@ def check_task_network(task_name, expected_network_name="dcos"):
                     assert "name" not in network_info, \
                         "Task {task} has network name when it shouldn't has status:{status}" \
                         .format(task=task_name, status=status)
+
+
+def get_endpoints(endpoint_to_get, package_name, correct_count):
+    endpoints, _, rc = shakedown.run_dcos_command("{} endpoints {}".format(package_name, endpoint_to_get))
+    assert rc == 0, "Failed to get endpoints on overlay network"
+    endpoints = json.loads(endpoints)
+    assert len(endpoints) == correct_count, "Wrong number of endpoints, got {} should be {}" \
+        .format(len(endpoints), correct_count)
+    return endpoints
+
+
+def check_endpoints_on_overlay(endpoints):
+    ip_addresses = [e.split(":")[0] for e in endpoints["address"]]
```

Comment:
My opinion is this is doing too much for one line.  It's nice when your python is so simpleminded, the line and exception tell you what's actually wrong.  Maybe it's less essential for test code, but I'm doubtful.